src/Generators/DocBlockGenerator.php

```diff
@@ -97,15 +108,14 @@ public function getGeneratedDocBlock()
      */
     protected function mergeGeneratedTagsIntoDocBlock($existingDocBlock)
     {
-        $docBlock = new DocBlock($this->removeExistingSupportedTags($existingDocBlock));
+        $docBlock = $this->docBlockFactory->create(($existingDocBlock ?: "/**\n*/"));
 
-        if (!$docBlock->getText()) {
-            $docBlock->setText('Class \\' . $this->className);
+        $summary = $docBlock->getSummary();
+        if (!$summary) {
+            $summary = 'Class \\' . $this->className;
```

Comment:
Can you write this as `sprintf('Class \\%s', $this->className)` please? I prefer concats to use sprintf in general, so if you see it elsewhere, please also update it :)